docs/backends/gcloud.rst

```diff
@@ -0,0 +1,190 @@
+Google Cloud Storage
+====================
+
+Usage
+*****
+
+This backend provides support for Google Cloud Storage using the
+library provided by Google.
+
+It's possible to access Google Cloud Storage in S3 compatibility mode
+using other libraries in django-storages, but this is the only library
+offering native support.
+
+By default this library will use the credentials associated with the
+current instance for authentication. To override this, see the
+settings below.
+
+
+Settings
+--------
+
+To use gcloud set::
+
+    DEFAULT_FILE_STORAGE = 'storages.backends.gcloud.GoogleCloudStorage'
+
+``GS_BUCKET_NAME``
+
+Your Google Storage bucket name, as a string.
+
+``GS_PROJECT_ID`` (optional)
+
+Your Google Cloud project ID. If unset, falls back to the default
+inferred from the environment.
+
+``GS_CREDENTIALS`` (optional)
+
+The OAuth 2 credentials to use for the connection. If unset, falls
+back to the default inferred from the environment.
+
+``GS_AUTO_CREATE_BUCKET`` (optional, default is ``False``)
+
+If True, attempt to create the bucket if it does not exist.
+
+``GS_DEFAULT_ACL`` (optional)
+
+If set to ``private`` changes uploaded file's Access Control List from the default permission ``public-read`` to give owner full control and remove read access from everyone else. 
```

Comment:
Perhaps list the list of the options here or reference to https://cloud.google.com/storage/docs/access-control/lists#predefined-acl.

In addition, the predefined ACL aliases for JSON API are different than XML API.

JSON API | XML API
--- | ---
private | private
bucketOwnerRead | bucket-owner-read
bucketOwnerFullControl | bucket-owner-full-control
projectPrivate | project-private
authenticatedRead | authenticated-read
publicRead | public-read
publicReadWrite | public-read-write